### PR TITLE
:poop: A great piece to start with

### LINKED ISSUE #1: nothing is there
needs fixing.

### PR DESCRIPTION
**Pull request by AnkeAnke**
There needs to be a start, and any start is better than none.
Also, top sneaky.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
 # Penultima Ideas
 
 Penultima ideas is the penultimate collection of penultima ideas.
+
+## Pawns
+
+### :poop: Not  pawn
```
Comment (Wumpf) [inline README.md]:
```suggestion
### :poop: Not a pawn
```

**Review by Wumpf — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AnkeAnke) [reply README.md]: A very good point, might have made that myself if I was sitting next to you and pointed the exact thing out.

Comment (gaetjen) [reply README.md]: Level 4 headers for single pieces pls

**Review by Wumpf — COMMENTED** [review]
@gaetjen @Gnaarf I'm ok with this pr, but I think it violates the current (weird) convention that most info is in `main.md`. Can we schedule a meeting to go over this a bit more? Thx.

File: README.md (hunk 1 above)
Comment (gaetjen) [reply README.md]: nevermind, this isn't the main.md

**Pull request merged**